source/renderer/app/stores/ProfileStore.ts

```diff
@@ -293,6 +306,22 @@ export default class ProfileStore extends Store {
     return this.getTermsOfUseAcceptanceRequest.result === true;
   }
 
+  @computed
+  get areAnalyticsAccepted(): boolean {
```

Comment:
```suggestion
  get analyticsAccepted(): boolean {
```

Reply:
Hey Shawn, we seem to be prefixing booleans with is/are, and it seems contradicting your previous comment: https://github.com/input-output-hk/daedalus/pull/2927#pullrequestreview-1018802865 Would you still like me to change it?

Reply:
Hehe. Yes maybe we can leave this one.